control-plane/config/crd/bases/consul.hashicorp.com_routeauthfilters.yaml

```diff
@@ -6,7 +6,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.11.3
```

Comment:
weird that this is the inverse of the change I commented on earlier 😕

Reply:
missed this during the rebasing